docs/API.md

```diff
@@ -121,7 +121,17 @@ The filters are a JSON object with `true`/`false` values. If not supplied, defau
 filters override their default values. The `compilerOptions` is used to pass extra arguments to the back end, and is
 probably not useful for most REST users.
 
-To force a cache bypass, set `bypassCache` in the root of the request to `true`.
+To force a cache bypass, `bypassCache` can be set. This accepts an enum value according to:
```

Comment:
What happens to thirdparty users setting this to true? Does the new api allow for that fallback?

Reply:
Oh just seen https://github.com/compiler-explorer/compiler-explorer/pull/5111/files#diff-e16897a5a4f4bbb218c2be258bb4908e073a4d3272181618dbe77c1cf559a234R92, I guess that's what backwards compat means here :)

Reply:
Yes! I went ahead and added some comments here and in the compiler request handler code.